modules/platforms/cpp/DEVNOTES.txt

```diff
@@ -63,39 +63,66 @@ Ignite expects for boost libraries and header to be found under default system p
 it is recommended to use package repository for your OS to install boost package for the
 development.
 
-Building on Linux and Mac OS X with CMake
+Building with CMake
 ----------------------------------
 
 Common Requirements:
-
- * GCC, g++, CMake >= 3.6 must be installed
+ * C++ compiler and SDK:
+  Linux and Mac OS X:
+   * clang >= 3.9 or gcc >= 3.6
+  Windows:
+   * Visual Studio 2010 or later
+   * Windows SDK 7.1 or later
+ * CMake >= 3.6 must be installed
```

Comment:
```suggestion
   * CMake >= 3.6 must be installed
```

Reply:
No, it's actually correct: CMake is a separate bullet-point and is not related exclusively to Windows - it's also used for Linux.